client/src/pages/component/BackgroundImageComponent/BackgroundImage.css

```diff
@@ -0,0 +1,50 @@
+.background-image-container {
+	background-image: url(../image/home-body.jpg) ;
+	background-repeat: no-repeat;
+    background-position: center center;
+	width: 100%;
+	height: 480px;
+	background-size: auto;
```

Comment:
I think the background size should be contained so it works on all devices and contains the picture

Reply:
Left it as auto

Reply:
cool